.vscode/scripts/run-seatservice.sh

```diff
@@ -14,57 +14,50 @@
 # SPDX-License-Identifier: Apache-2.0
 
 echo "#######################################################"
-echo "### Running Seatservice                             ###"
+echo "### Running Seat Service                            ###"
 echo "#######################################################"
 
 ROOT_DIRECTORY=$( realpath "$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )/../.." )
 source $ROOT_DIRECTORY/.vscode/scripts/exec-check.sh "$@" $(basename $BASH_SOURCE .sh)
 
-SEATSERVICE_VERSION=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .seatservice.version | tr -d '"')
-SEATSERVICE_PORT='50051'
-SEATSERVICE_GRPC_PORT='52002'
-sudo chown $(whoami) $HOME
+SEATSERVICE_IMAGE=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .seatservice.image | tr -d '"')
+SEATSERVICE_TAG=$(cat $ROOT_DIRECTORY/prerequisite_settings.json | jq .seatservice.version | tr -d '"')
 
-SEATSERVICE_ASSET_FOLDER="$ROOT_DIRECTORY/.vscode/scripts/assets/seatservice/$SEATSERVICE_VERSION"
-#Detect host environment (distinguish for Mac M1 processor)
-if [[ `uname -m` == 'aarch64' || `uname -m` == 'arm64' ]]; then
-    echo "Detected ARM architecture"
-    PROCESSOR="aarch64"
-    SEATSERVICE_BINARY_NAME="bin_vservice-seat_aarch64_release.tar.gz"
-else
-    echo "Detected x86_64 architecture"
-    PROCESSOR="x86_64"
-    SEATSERVICE_BINARY_NAME="bin_vservice-seat_x86_64_release.tar.gz"
-fi
-SEATSERVICE_EXEC_PATH="$SEATSERVICE_ASSET_FOLDER/$PROCESSOR/target/$PROCESSOR/release/install/bin"
-
-if [[ ! -f "$SEATSERVICE_EXEC_PATH/val_start.sh" ]]
+RUNNING_CONTAINER=$(docker ps | grep "$SEATSERVICE_IMAGE" | awk '{ print $1 }')
```

Comment:
Please add Help() function to explain the parameters

Maybe we need to add verification in front to make sure that parameter is provided and in case of failure - clear error message shall be shown

Reply:
See above